brave/brave-core
# Update rewards onboarding panel in welcome UI

Resolves https://github.com/brave/brave-browser/issues/13119

This PR updates the design of the rewards panel within the welcome UI:

- Terms of service and privacy policy links have been added in a smaller font.
- The graphic has been updated.
- The explainer text has been updated.
- When the primary button is clicked, the user is taken to the rewards settings page and automatically opted-in. They will no longer be shown an onboarding modal.

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.

## Test Plan:

(Both light and dark theme should be tested.)

- Open browser with clean profile.
- Navigate through welcome panels until the rewards panel appears.
  - Verify that terms of service and privacy policy links are displayed.
  - Verify that graphic has been updated to match [designs](https://www.figma.com/file/0dbMYs8Wp8W3RgrG1yTdki/RE-1.0?node-id=297%3A21196).
  - Verify that the terms of service link opens a new tab to https://basicattentiontoken.org/user-terms-of-service/
  - Verify that privacy policy link opens a new tab to https://brave.com/privacy/browser/#rewards
- Click the "Start using Rewards" button.
  - Verify that a new tab is opened to the rewards settings page.
  - Verify that the onboarding modal is not displayed.
  - Verify that ads and AC are enabled.
